### ISSUE TITLE
Harry potter chamber of secrets does not boot in an extremely way

### ISSUE THREAD

**Issue by Jordanesquivez** [jpd002/Play-#583]
Yeah so I've tried this game and the result was really awful its just stuck on black screen with no DC/f and fp/s... Maybe a loop I guest but nothing is moving

**Comment by Kygies27** [comment]
Yeah, I've seen that problem before (specifically in Duel Masters LTD). I'm not a debugger or anything but it got fixed pretty quick in the next update (that may have been luck though).

Pretty sure I've seen it around here before that they're working on compatability before speed so don't worry too much about it  (y)

**Comment by paulaz2** [comment]
Similar to harry potter and the half blood prince

**Comment by jpd002** [comment]
Got the game to boot, it was because of an issue with command line parameters. The game still doesn't work really well though.

![capture](https://user-images.githubusercontent.com/6760433/29746556-e3463a86-8aab-11e7-8428-f0374152b4e9.PNG)
![capture2](https://user-images.githubusercontent.com/6760433/29746555-e3453eba-8aab-11e7-89c0-5fe9a8046ed4.PNG)

Here's the branch for those who might be curious: https://github.com/jpd002/Play-/tree/harrypotter

**Comment by paulaz2** [comment]
Your doing a great job jpd002 always have believed in you....so did you test harry potter and half blood prince?

**Comment by jpd002** [comment]
I haven't tested it, I'll see if I can get my hands on that game.

**Comment by paulaz2** [comment]
Thanks.

**Comment by jpd002** [comment]
I merged the fix in master yesterday. If you'd like to try Harry Potter and the Half Blood Prince, you can use these builds:
https://s3.us-east-2.amazonaws.com/playbuilds/06a1cc84/Play-0.30-64.exe
https://s3.us-east-2.amazonaws.com/playbuilds/06a1cc84/Play-release.apk

**Comment by paulaz2** [comment]
Is it working well?

**Comment by jpd002** [comment]
The Chamber of Secrets doesn't work that well, but it doesn't mean The Half Blood Prince will be as bad.

**Comment by paulaz2** [comment]
Well have not tested it don't have the ROM anymore, well I will try to get my hands on it and see how it works with the emu. Thanks for the reply

**Comment by atomic83GitHub** [comment]
These improvement made VampireNight to go ingame!

**Comment by jpd002** [comment]
@paulaz2 Thanks, I'll get the game too just in case there's other issues with it.

@atomic83GitHub Nice! I know this improves other games such as Hot Shots Golf Fore (even though it still doesn't go in game).

**Comment by paulaz2** [comment]
Well @jpd002 downhill domination still studied at loading screen.what seems to be the problem?

**Comment by atomic83GitHub** [comment]
Ace combat 4 now go ingame!!

![image](https://user-images.githubusercontent.com/26351275/30201711-0d09cfac-947c-11e7-9307-1f3134d98fdb.png)

**Comment by jpd002** [comment]
@paulaz2 Downhill Domination is using unimplemented system calls, but even with the system calls implemented, it still doesn't load completely.

@atomic83GitHub Great news! Thanks for reporting!

**Comment by paulaz2** [comment]
@jpd002 so whats the step forward for downhill domination?

**Comment by jpd002** [comment]
@paulaz2 Gotta put some time in trying to fix it. I don't know when I'll be able to do so, I can't guarantee anything. Also, let's keep this thread focused on Harry Potter: Chamber of Secrets.

**Comment by paulaz2** [comment]
@jpd002 sorry yea that's what its was created for
